packages/react-integration/demo-app-ts/package.json

```diff
@@ -11,8 +11,8 @@
     "@patternfly/react-core": "workspace:^",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-router": "^5.3.4",
-    "react-router-dom": "^5.3.4"
+    "react-router": "^7.2.0",
+    "react-router-dom": "^7.2.0"
```

Comment:
`devDependencies` has references to `@types/react-router` and `@types/react-router-dom` at versions `5.1.20` and` 5.3.3`, respectively. thought this was odd to be so far behind, and after a bit of research it seems that starting from version 6, react-router and react-router-dom each include their own TypeScript type definitions, so the tldr is that these can both be removed from `devDependencies`.